bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/example_cards/HToZATo2L2B_500p00_300p00_1p50_ggH_TuneCP5_13TeV_pythia8/HToZATo2L2B_500p00_300p00_1p50_ggH_TuneCP5_13TeV_pythia8_run_card.dat

```diff
@@ -23,7 +23,7 @@
 # Number of events and rnd seed                                      *
 # Warning: Do not generate more than 1M events in a single run       *
 #*********************************************************************
-  5000	= nevents ! Number of unweighted events requested 
+  20000	= nevents ! Number of unweighted events requested 
```

Comment:
Why do you need to increse?

Reply:
We are relying on DNN for signal extraction and due to low acceptance and large fraction of negative weight mainly for bbH process @ NLO :
an example of one generated sample 
```After filter: final fraction of events with negative weights = 4.003e-01 +- 1.961e-03```
For ggH @ LO we are not facing any problem for all samples i checked i see after showering with pythia8:
```Filter efficiency (taking into account weights)= (1000) / (1000) = 1.000e+00 +- 0.000e+00```
So, we are asking for large number of events to be generated indeed !

Reply:
This seems a general misunderstanding. This is only relevant for the pilot run and has nothing to do with central production or filters etc .

Reply:
As Alexander said, the number here won't be used as the actual nevents that you request for central production.
It is only used as a pilot run for small nevents (2-3K), to check whether your gridpack can generate LHE files without crashing while making gridpacks.

Reply:
[done](https://github.com/kjaffel/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/example_cards/HToZATo2L2B_500p00_300p00_20p00_bbH4F_TuneCP5_13TeV-amcatnlo_pythia8/HToZATo2L2B_500p00_300p00_20p00_bbH4F_TuneCP5_13TeV-amcatnlo_pythia8_run_card.dat#L34). I am able to generate a valid LHE file.